src/modules/UI/components/FlipInput/styles.js

```diff
@@ -5,6 +5,7 @@ import THEME from '../../../../theme/variables/airbitz'
 
 export const styles = StyleSheet.create({
 
+
   // Main Flip Input Styles
```

Comment:
Please refrain from using comment inside objects

Reply:
That was there even before I edited it, did not put it, but will remove, the blank likes I did